### DISCUSSION TITLE
please help me

### DISCUSSION THREAD

**Discussion by samaremad** [serenity-is/Serenity#5344] [Help]
Hello everyone

i just learning serenity and I am already make the admin panel but now I want to make the interface and connect it with this panel could any one help me with that

**Comment by dfaruque** [comment]
It is not clear. what kind of help you need?

**Reply by samaremad** [reply]
first : thank you for your reply 
second : I made the dashboard and downloaded bootstrap for front end page which I want to connect it with the dashboard and don't know how to do that and where can I added the code of that page

**Comment by dfaruque** [comment]
If you want to customize the dashboard then you may look into the folder "Modules/Common/Dashboard"

**Reply by samaremad** [reply]
no sir i already complete the dashboard
 I want to create home page which the users (not the admin ) can reached 
so I want to add link in the dashboard that link open the page which I want to create

**Comment by dfaruque** [comment]
simply put the link to that page in your dashboard. in the demo dashboard you have already have the links. eg. in the "Open Orders" box you have a link "More Info" https://serenity.is/demo/

**Comment by dfaruque** [comment]
to change dashboad page with some condition you can do like following
```
public class DashboardController : Controller
    {
        [Authorize, HttpGet, Route("~/")]
        public ActionResult Index()
        {
            var user = Authorization.UserDefinition as UserDefinition;

            if (user is null)
            {
                Response.Redirect("~/Account/Login");
                return null;
            }
            else
            {
                var userType = user?.UserType;
                var model = GetDashboardModel();

                if (userType == UserType.NewConnectionApplicant)
                {
                        return View(MVC.Views.Common.Dashboard.DashboardPublic);
                }
                else if (userType == UserType.Consumer)
                {
                    return View(MVC.Views.Common.Dashboard.Consumer.DashboardConsumer);
                }
                else
                {
                    return View(MVC.Views.Common.Dashboard.DashboardIndex, model);
                }
            }
        }
```

**Reply by samaremad** [reply]
ok but now i just want to add link without condition i dont know where suppose i added the link and where can i write the script of that page
